### ISSUE TITLE
Temperatures well above the adiabatic temperature

### ISSUE THREAD

**Issue by gforney** [firemodels/fds#1237]
```
Application Version:FDS5.5.1
SVN Revision Number:
Compile Date:June 2010
Operating System:Linux 64 bit

Hi,
I am trying to model a jet flow and I am conistently getting temperatures that are
well above the adiabatic temperature for propane (gas I am trying to model). 
The adiabatic temperature of propane is 1990C or there about depending on the literature,
the temperatures I am getting are above 2300C. 
I understand that high velocity flows are not handed well by FDS but when I try to
reduce the jet velocity down to less than 30m/s I am still getting the same problem.
Please see the attached FDS input file.
Kind regards,
KP Wolnicki
```

Original issue reported on code.google.com by `CWolnicki` on 2010-11-20 15:33:44

<hr>
- _Attachment: [jet.fds](https://storage.googleapis.com/google-code-attachments/fds-smv/issue-1266/comment-0/jet.fds)_

**Comment by gforney** [comment]
```
The FDS combustion model will burn whatever fuel and oxygen are deemed capable of mixing
together over a timestep. You have one cell across the inlet to your jet.  How well
do you think you are resolving the flow field and entrainment?  My guess is poorly.
 FDS is an LES code, not a RANS code.  You need to have sufficient grid resolution
to resolve the major flow structures and one grid cell across the inlet means you cannot
even resolve a singe eddy within the near field plume. I suggest you experiment with
much higher grid resolutions near the jet (e.g. perform a grid study to demonstrate
you are predicting the jet flows correctly).  If you still see excessive temperatures
regardless of grid refinement, then report back.
```

Original issue reported on code.google.com by `drjfloyd` on 2010-11-20 22:17:28
- **Labels added**: Application-FDS

**Comment by gforney** [comment]
```
Thank you for your prompt reply. I see and agree with your point that the flow within
the domain may not be realistic as the grid is much bigger than the size of the inlet.

The problem affecting the flow is likely to influence the transfer of heat but should
not raise the temperature of combustion above the adiabatic temprature as the latter
is the theoretical maximum temprature which is dependant on the the fuel and already
assumes the most advantagous, stoichometirc mixture of fuel and air. I do agree that
as a result of problems with the flow there may be accumulation of heat within a cell
but this should result in increase of residuals and possible numerical instabilities
rather than increase of temperature above the adiabatic temperature. After all, increase
of temperature as a result of mixing two cooler fluid flows would be against the principles
of thermodynamics.  
I tried a range of mesh sizes and observe that flow is much affected by the grid. Unfortunately,
I observe that the maximum temperature does not come down towards the adiabatic temperature
as I reduce the grid size.
Did I understand your previous comment correctly? If so, is there any other way that
comes to mind to solve this problem.
I would much appreciate your help. Kind regards,
KP Wolnicki

```

Original issue reported on code.google.com by `CWolnicki` on 2010-11-20 22:53:56

**Comment by gforney** [comment]
```
All of what you say is true if you are doing a Direct Numerical Simulation with finite
rate chemistry. You are doing LES with infinitely fast chemistry. By not resolving
the flame sheet, there is now the possibility that too much fuel and oxygen will find
itself within one grid cell at a given time step and burn rapidly, producing higher
than theoretical temperatures in certain regions of the flow field. The assumptions
being made in your calculation are those we use for fires on grids of 10s of cm. But
in your case, you have to consider every single assumption and decide if it is appropriate.

```

Original issue reported on code.google.com by `mcgratta` on 2010-11-21 00:43:26

**Comment by gforney** [comment]
```
Thank you very much for your invaluable comment.
Kind regards,
KP Wolnicki
```

Original issue reported on code.google.com by `CWolnicki` on 2010-11-22 07:39:08

**Comment by gforney** [comment]
```
Closing this issue as it seems to have been resolved by comment #3
```

Original issue reported on code.google.com by `drjfloyd` on 2011-08-10 17:46:52
